### DISCUSSION TITLE
Custom shape divider + position, rotate and width options

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#21537] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

The shape divider is currently very limited. We can only use default shapes and change their color.

### Describe the solution you'd like

please develop shape divider and keep the following options in mind for it.

1. width
2. rotate
3. Custom position : https://prnt.sc/vQ8CeClbThkO
4. Custom shape divider

**Comment by astralbath** [comment]
I came here to look for exactly this. Hard same on this feature request.

**Comment by netmedia84** [comment]
**Same request:** It should be possible to add custom separator shapes in SVG format quite easily, as an additional feature to be added in the Elementor menu, similar to fonts, icons, and custom codes.
I tested a plugin that allows the addition of custom separator shapes in SVG.
https://github.com/MarieComet/elementor-custom-shapes
However, this plugin hasn’t been updated for 2 years, and you still need to manually edit the SVG code for compatibility.
Separator shapes are a very nice feature but currently quite limited.

**Comment by alanstainerPMW** [comment]
I was about to request the very same thing and thought I had better see if anyone else had requested it already. Good job I did! It would be great if we could upload a custom SVG and maybe have some additional options, for example repeating a shape to create a pattern.

**Comment by netmedia84** [comment]
Hi, there’s an official solution in Elementor with the theme.
See here: https://developers.elementor.com/docs/hooks/shapes/
